charts/standard-application-stack/templates/mariadb-py-dba.yaml

```
@@ -0,0 +1,72 @@
+{{- if (and .Values.mariadb .Values.mariadb.enabled) }}
+{{- if (and .Values.mariadb.extraUsers .Values.mariadb.extraUsers.enabled) }}
+---
+{{- $data := dict "Release" $.Release "Chart" $.Chart "Values" $.Values "component" .name }}
+apiVersion: {{ include "common.capabilities.job.apiVersion" $ }}
+kind: Job
+metadata:
+  name: {{ include "mintel_common.fullname" $data }}
+  labels: {{ include "mintel_common.labels" $data | nindent 4 }}
+  annotations:
+    {{- include "mintel_common.commonAnnotations" $ | nindent 4 }}
+    {{ if .argo }}
+    {{ include "mintel_common.argoAnnotations" .argo | nindent 4 }}
+    {{ end }}
+    helm.sh/chart: {{ include "mintel_common.chart" $ }}
+  namespace: {{ $.Release.Namespace }}
+spec:
+  ttlSecondsAfterFinished: 600
+  template:
+    spec:
+      {{- include "mintel_common.imagePullSecrets" $ | nindent 6 }}
+      securityContext:
+        runAsUser: 1000
```

This shouldn't be hard coded. I'd recommend something like
```suggestion
      {{- with .Values.mariadb.extraUsers.podSecurityContext | default $.Values.podSecurityContext }}
      securityContext: {{- toYaml . | nindent 8 }}
      {{- end }}
```

The test snapshot file should show at least
```
securityContext:
  runAsUser: 1000
  runAsNonRoot: true
```
Otherwise this pod won't pass the coming Kyverno checks.

---

the reason it's hard coded is because a user should never have to change the security context, given that the same image is always being run in this job.

What do you think about just adding `runAsNonRoot` here?
```suggestion
      securityContext:
        runAsUser: 1000
        runAsNonRoot: true
```

---

I've gone ahead with my suggestion above in the new revision, not against having it in the values if that would make more sense.

---

Yeah, alright.